Gemfile.lock

```diff
@@ -24,13 +24,13 @@ GEM
     diff-lcs (1.3)
     docile (1.1.5)
     highline (2.0.3)
-    i18n (1.11.0)
+    i18n (1.12.0)
```

Comment:
Just in case, I double checked this and looks safe https://github.com/ruby-i18n/i18n/releases